### PR TITLE
[feat]:filter functionality

### PR DESCRIPTION
**Pull request by ultronjs**

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is looking good

File: src/components/FilterModal.jsx [hunk 1]
```diff
@@ -1,9 +1,117 @@
...
+            }
+          </div>
+        </div>
+        <div class="modal_footer">
+          {/* <button
+            onClick={() => setShowFilterModal(false)}
+            class="btn btn_dark modal_cancel_btn"
+          >
+            Cancel
+          </button> */}
```
Comment (sushil-mod) [inline src/components/FilterModal.jsx:101]: you can remove this commented code if not in use

**Review by sushil-mod — APPROVED** [review]
Filter Working fine . overall good code

**Comment by ultronjs** [comment]
Merging after 2 approval

**Pull request merged**